apps/new/widget/page/projects/CatalogImport.jsx

```diff
@@ -0,0 +1,193 @@
+const indexer = "https://nearcatalog.xyz/wp-json/nearcatalog/v1";
+const projects = {};
+const query = "";
+
+const CardSkeletonContainer = styled.div`
+  @keyframes loadingSkeleton {
+    0% {
+      opacity: 0.8;
+    }
+    50% {
+      opacity: 0.3;
+    }
+    100% {
+      opacity: 0.6;
+    }
+  }
+
+  display: flex;
+  flex-direction: column;
+  height: 447px;
+  width: 100%;
+  max-width: 400px;
+  border-radius: 12px;
+  background: var(--bg-color, #23242b);
+  color: var(--text-color, #fff);
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  animation-name: loadingSkeleton;
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
+`;
+
+const HeaderSkeleton = styled.div`
+  display: block;
+  width: 100%;
+  height: 168px;
+  background: #eee;
+`;
+
+const TitleSkeleton = styled.div`
+  width: 120px;
+  height: 24px;
+  background: #eee;
+  margin-left: 24px;
+  margin-top: 24px;
+`;
+
+const DescriptionSkeleton = styled.div`
+  width: 83%;
+  height: 48px;
+  background: #eee;
+  margin-left: 24px;
+  margin-top: 24px;
+`;
+
+const TagSkeleton = styled.div`
+  background: #eee;
+  border-radius: 4px;
+  height: 34px;
+  width: 110px;
+  margin: 24px;
+`;
+
+const CardSkeleton = () => (
+  <CardSkeletonContainer>
+    <HeaderSkeleton />
+    <TitleSkeleton />
+    <DescriptionSkeleton />
+    <TagSkeleton />
+  </CardSkeletonContainer>
+);
+
+query = fetch(indexer + "/projects");
+if (!query || !query.body) {
+  return "Loading...";
+}
+projects = query.body;
+
+if (!projects) {
+  return "No projects found";
+}
+
+const Container = styled.div`
+  .error-bg {
+    background-color: #4d4d4d;
+    color: white;
+  }
+  max-width: 100%;
+`;
+
+const [filteredProjects, setFilteredProjects] = useState(projects);
```

Comment:
Looks awfully familiar to PotlockImport 

Maintaining the same code in two different places... 😮‍💨

Reply:
haha, it tends to be similar because it's just another way to do the same thing